src/index.js

```diff
@@ -42,7 +42,7 @@ import Item from 'src/views/Items/Item';
 import Items from 'src/views/Items/Items';
 import Statistic from 'src/views/Statistic/Statistic';
 
-export default {
+const stardust = {
```

Comment:
:+1: removed the `karma.conf.js` register hook and added scripts for running karma with babel-node.  It looks like babel-node requires the filename be named `karma.conf.babel.js`, so I left that as is.